notes/returned-mail.vbs

```diff
@@ -336,65 +357,73 @@ ELSE
 CALL write_bullet_and_variable_in_CASE_NOTE("METS case number", METS_case_number)
+CALL write_bullet_and_variable_in_CASE_NOTE("Confirmed address is correct", received_error_confirmation)
 IF mailing_address_confirmed = "YES" THEN  'Address Detail
-	CALL write_variable_in_CASE_NOTE("* Returned mail received from: " & mail_line_one)
-	If mail_line_two <> "" Then CALL write_variable_in_CASE_NOTE("                             " & mail_line_two)
-	CALL write_variable_in_CASE_NOTE("                               " & mail_city_line & ", " & mail_state_line & " " &   mail_zip_line)
+	CALL write_variable_in_CASE_NOTE("* Returned mail received from (mailing): " & mail_line_one)
+	If mail_line_two <> "" Then CALL write_variable_in_CASE_NOTE("                                        " & mail_line_two)
+	CALL write_variable_in_CASE_NOTE("                                        " & mail_city_line & ", " & mail_state_line & " " &   mail_zip_line)
 END IF
 IF residential_address_confirmed = "YES" THEN
-	CALL write_variable_in_CASE_NOTE("* Returned mail received from: " & resi_addr_line_one)
-	If resi_addr_line_two <> "" Then CALL write_variable_in_CASE_NOTE("                               " & resi_addr_line_two)
-	CALL write_variable_in_CASE_NOTE("        		                 " & resi_addr_city & ", " & resi_addr_state & " " & resi_addr_zip)
+	CALL write_variable_in_CASE_NOTE("* Returned mail received from (residential): " & resi_addr_line_one)
+	If resi_addr_line_two <> "" Then CALL write_variable_in_CASE_NOTE("                                             " & resi_addr_line_two)
+	CALL write_variable_in_CASE_NOTE("                                             " & resi_addr_city & ", " & resi_addr_state & " " & resi_addr_zip)
 END IF
 IF homeless_addr = "Yes" Then Call write_variable_in_CASE_NOTE("* Household reported as homeless")
 IF reservation_addr = "Yes" THEN CALL write_variable_in_CASE_NOTE("* Reservation " & reservation_name)
-Call write_bullet_and_variable_in_CASE_NOTE("Address Detail", notes_on_address)
-
+CALL write_bullet_and_variable_in_CASE_NOTE("Address detail", notes_on_address)
+CALL write_bullet_and_variable_in_case_note("Verification(s) requested", verifications_requested)
+CALL write_bullet_and_variable_in_case_note("Verification(s) due", due_date)
 IF ADDR_actions = "forwarding address provided" THEN
 	CALL write_variable_in_CASE_NOTE("* Forwarding address was on returned mail")
 	CALL write_variable_in_CASE_NOTE("* Mailing address updated:  " & new_addr_line_one)
 	If new_addr_line_two <> "" Then CALL write_variable_in_CASE_NOTE("                            " & new_addr_line_two)
 	CALL write_variable_in_CASE_NOTE("                            " & new_addr_city & ", " & new_addr_state & " " & new_addr_zip)
-	CALL write_variable_in_case_note("* Client must be provided 10 days to return requested verifications")
+	CALL write_variable_in_case_note("* Resident must be provided 10 days to return requested verifications")
```

Comment:
Add period

Reply:
We should add this to closing documentation as it is not consistent

Reply:
CALL write_variable_in_CASE_NOTE ("* ECF reviewed for requested verifications.") VS.
	CALL write_variable_in_CASE_NOTE("* Date verification(s) requested: " & date_verifications_requested) - do you want them added to all ?